src/libostree/ostree-sysroot-upgrader.h

```diff
@@ -85,7 +85,8 @@ gboolean ostree_sysroot_upgrader_check_timestamps (OstreeRepo     *repo,
 
 typedef enum {
   OSTREE_SYSROOT_UPGRADER_PULL_FLAGS_NONE = 0,
-  OSTREE_SYSROOT_UPGRADER_PULL_FLAGS_ALLOW_OLDER = (1 << 0)
+  OSTREE_SYSROOT_UPGRADER_PULL_FLAGS_ALLOW_OLDER = (1 << 0),
+  OSTREE_SYSROOT_UPGRADER_PULL_FLAGS_SYNTHETIC = (1 << 1) /* Don't actually do a pull, just check timestamps/changed */
```

Comment:
The equivalent of `RPMOSTREE_TRANSACTION_DEPLOY_FLAG_NO_PULL_BASE`?  So...yeah, I guess we could do that, but I like keeping the timestamp check in there, and I'd argue we should probably do the same in rpm-ostree.

What if e.g. someone did a raw `ostree pull --commit-metadata-only` or something, which *won't* do timestamp checks and hence could be more easily MITM'd - then later an admin does `ostree admin upgrade --deploy-only`, we'd be skipping the timestamp check.